issuesync.rb

```diff
@@ -226,6 +229,14 @@ def updated_at() Time.parse(data['updated_at']) end
     def has_comments?() !data['comments'].zero? end
     def open?() data['state'] == 'open' end
     def closed?() !open? end
+
+    def labels()
+      s = ""
+      for label in data['labels']
+        s << "  _" + label['name'] + "_"
```

Comment:
Could we just have this method return an array of labels, or empty array if no labels? Then, the formatter (such as `format_body`) should be in charge of the format of the output.

I'm not so sure about formatting labels by surrounding them with underscores. Maybe square brackets would be better.